template/src/config/ui/typography.ts

```diff
@@ -0,0 +1,14 @@
+import { StyleSheet, TextStyle } from 'react-native'
```

Comment:
How is typography related to "Networking improvements"? Why is it in this PR? I think `typography.ts` and `DemoTextInput.tsx` could be in a separate PR.

I miss some information about what is this file and how to use it. Maybe it makes sense to create another Issue or PR for describing it?

Reply:
it happend to be here, cause I needed some textInputs to showcase the login. I'll add a comment to this file. it was meant to hold app text styles.